source/ota_mqtt.c

```diff
@@ -628,7 +628,7 @@ static uint32_t buildStatusMessageReceiving( char * pMsgBuffer,
     received = numBlocks - pOTAFileCtx->blocksRemaining;
 
     /* Output a status update once in a while. */
-    if( ( received % otaconfigOTA_UPDATE_STATUS_FREQUENCY ) == 0U )
+    if( ( received == 1 ) || ( ( received % otaconfigOTA_UPDATE_STATUS_FREQUENCY ) == 0U ) )
```

Comment:
Will the function be called for each block received? Can we add/update unit tests for the changes?

Reply:
Yes, the function is called for every block received. Added a test case in #425.

Reply:
I see PR is closed. Is the plan to update this PR with the changes?

Reply:
I see PR is closed. Is the plan to update this PR with the changes?